utils/Felt.go

```diff
@@ -87,3 +90,54 @@ func FeltArrToBigIntArr(f []*felt.Felt) []*big.Int {
 	}
 	return bigArr
 }
+
+const SHORT_LENGTH = 31
```

Comment:
As we will unify both functions, let's put this const inside the "ByteArrToFelt" function

Reply:
Done